warehouse/templates/manage/project/settings.html

```diff
@@ -60,7 +60,10 @@ <h3>{% trans %}API tokens{% endtrans %}</h3>
     <h3>{% trans %}Project description and sidebar{% endtrans %}</h3>
     <p>
       {% trans project_name=project.name, pyproject_args_href='https://packaging.python.org/tutorials/packaging-projects/#creating-pyproject-toml', twine_docs_href='https://twine.readthedocs.io/', distribution_href='https://packaging.python.org/guides/section-build-and-publish/' %}
-        To set the '{{ project_name }}' description, author, links, classifiers, and other details for your next release, use the <a href="{{ pyproject_args_href }}" rel="noopener" target="_blank"><code>setup()</code> arguments in your <code>setup.py</code> file</a>.
+        To set the '{{ project_name }}' description, author, links, classifiers,
+        and other details for your next release, use the
+        <a href="{{ pyproject_args_href }}" rel="noopener" target="_blank"><code>project</code>
+        metadata fields in your <code>pyproject.toml</code> file</a>.
```

Comment:
NB: The underlying link here already changed to documenting PEP 621 `project` metadata, so all I've done here is update the link text 🙂